### ISSUE TITLE
Installation warning

### ISSUE THREAD

**Issue by fundatillus** [gpodder/mygpoclient#17]
Did a fresh install of gPodder on an Ubuntu 20.04 box and received this error during installation: 
```
Setting up python3-mygpoclient (1.8-3) ... 
/usr/lib/python3/dist-packages/mygpoclient/locator.py:320: SyntaxWarning: "is" with a literal. Did you mean "=="? if type is 'device':
/usr/lib/python3/dist-packages/mygpoclient/locator.py:325: SyntaxWarning: "is" with a literal. Did you mean "=="? if type is 'podcast':
/usr/lib/python3/dist-packages/mygpoclient/locator.py:330: SyntaxWarning: "is" with a literal. Did you mean "=="? if type is 'episode':
```

**Comment by auouymous** [comment]
Fixed in 1.9.

**Comment by fundatillus** [comment]
Great, thanks!
